docs/query.rst

```diff
@@ -286,6 +286,46 @@ Specially, you can filter date part with one of following, note that current onl
     teams = await Team.filter(created_at__month=12)
     teams = await Team.filter(created_at__day=5)
 
+In the PostgreSQL, you can use the ``contains``, ``contained_by`` and ``filter`` options in ``JSONField``:
+
+.. code-block:: python3
+
+    class JSONModel:
+        data = fields.JSONField()
+
+    await JSONModel.create(data=["text", 3, {"msg": "msg2"}])
+    obj = await JSONModel.filter(data__contains=[{"msg": "msg2"}]).first()
+
+    await JSONModel.create(data=["text"])
+    await JSONModel.create(data=["tortoise", "msg"])
+    await JSONModel.create(data=["tortoise"])
+
+    objects = await JSONModel.filter(data__contained_by=["text", "tortoise", "msg"])
+
+.. code-block:: python3
+
+    class JSONModel:
+        data = fields.JSONField()
+
+    await JSONModel.create(data={"breed": "labrador",
+                                 "owner": {
+                                     "name": "Boby",
+                                     "last": None,
+                                     "other_pets": [
+                                         {
+                                             "name": "Fishy",
+                                         }
+                                     ],
+                                 },
+                             })
+
+    obj = await JSONModel.filter(data__filter={"breed": "labrador"}).first()
```

Comment:
In Django, the query is `JSONModel.filter(data__breed="labrador").first()`, also other cases, I'm not sure whether we can do like that?

Reply:
In `tortoise-orm` , filters are a dictionary and these are implemented in `filters.py` and `get_filters_for_field` function. It returns a dictionary that `keys` are filter name and I decided to implement `data__filter` key. I think we need more changes in `tortoise-orm` MetaModel and Queryset to implement like Django orm. So this way has less change rather than Django orm type.
In addition I found out there was a way to implement the feature only for `postgresql`. I will push new changes.